src/javascript/app/pages/trade/event.js

```diff
@@ -367,6 +367,7 @@ const TradingEvents = (() => {
          */
         const low_barrier_element = getElementById('barrier_low');
         low_barrier_element.addEventListener('input', CommonTrading.debounce((e) => {
+            Barriers.validateBarrier();
```

Comment:
why not pass if it's low or high calling validateBarrier here, instead of checking if low or high value in store is different from input field?

Reply:
Because We can generalize the function for uses of validation in both `barriers.jsx` and `event.js`.

- validateBarrier() is used in `display()` function in `barriers.jsx` (no e.target). 
    - there is no `e.target.value` in display() -  `barrier.js (line 76 and 146)` .

Reply:
but in barrier.js it's not user input, so we can just default to first_time, no?